tools/build/third_party/perfetto/perfetto.BUILD

```diff
@@ -25,6 +25,7 @@ _COPTS_BASE = cc_copts() + [
     "-DNDEBUG",
 ] + select({
     "@gapid//tools/build:windows": ["-D__STDC_FORMAT_MACROS"],
+    "@gapid//tools/build:linux": ["-Wno-error=class-memaccess"],
```

Comment:
Add a TODO comment referring to issue:
`TODO(#3099): remove Wno-error=class-memaccess once #3099 is fixed`